index.bs

```diff
@@ -2541,6 +2555,9 @@ This [=registration extension=] and [=authentication extension=] enables use of
 : Client processing
 :: None, except default forwarding of client argument to authenticator argument.
 
+: Client data
+:: Returns a JSON string containing the base64url encoding of the authenticator data
+
```

Comment:
ClientData (which must be hashed and fed into the authenticator for signature) cannot depend on AuthenticatorData (which comes out of the authenticator after the signature has been computed). TShould be "none".